Sources/XS2AiOS/XS2AViewController.swift

```diff
@@ -118,10 +123,12 @@ public class XS2AViewController: UIViewController, UIAdaptivePresentationControl
 				}
 			})
 			self.stackView.layoutIfNeeded()
+            self.preferredContentSize = .zero
```

Comment:
Same question here, is default behaviour affected? If yes this should go into an `if` for this scenario.

Reply:
Indentation needs to be tabs to align with rest.